Makefile

```diff
@@ -211,18 +211,22 @@ gitserver:
 
 .PHONY: e2e-test-clean
 e2e-test-clean:
-	$(KUBECTL) delete -k tests/e2e/manifests --ignore-not-found=true
-	$(KUBECTL) delete -f tests/e2e/manifests/numaplane-ns.yaml --ignore-not-found=true
+	$(KUBECTL) delete -n numaplane-e2e isbsvc --all
+	$(KUBECTL) delete -n numaplane-e2e pipelines --all
+	$(KUBECTL) delete -n numaplane-e2e cm --all
+	$(KUBECTL) delete -n numaplane-e2e all --all
```

Comment:
I guess you're no longer including these resources. What happens if you just have `$(KUBECTL) delete -n numaplane-e2e all --all` instead of the 3 lines above?

Reply:
oh wait, I think I'm wrong about that...looks like you are using numaflow resources...sorry

Reply:
That's weird about the ConfigMaps. Did you find any corroboration of that on the internet by chance?

> I was unsure about the whole installing the CRDs question for the CI

But you're still going to need it in the CI if it's in the one test case, right?

Reply:
Also, why do the isbsvc's and pipelines need to be deleted first? Is it because you're installing a Numaflow Controller, and those need to have their finalizer removed before the Controller is deleted?

Reply:
Good find! That's crazy, though. Can you add comments to indicate why you're deleting the way you are in case someone's wondering later? Maybe also link to the comment wrt the ConfigMap?

Reply:
Will do!